### PR TITLE
[ETFE-2533] - EIS integration for CAM

### PR DESCRIPTION
**Pull request by LewisMitchell**

### CODE REVIEW

File: app/uk/gov/hmrc/emcstfe/models/request/SubmitCreateMovementRequest.scala [hunk 1]
```diff
@@ -74,4 +79,22 @@ case class SubmitCreateMovementRequest(body: CreateMovementModel)
...
         body.consignorTrader.countryCode
     }
+
+  override def eisXMLBody(): String = {
+    withEisMessage(
+      body = body,
+      messageNumber = messageNumber,
+      messageSender = messageSender,
+      messageRecipient = messageRecipient,
+      messageIdentifier = draftId
```
Comment (grahampaulcook) [inline app/uk/gov/hmrc/emcstfe/models/request/SubmitCreateMovementRequest.scala:89]: Even though we won't be using ChRIS for the submission, I wonder if it's still worth populating the `draftId` into  https://github.com/hmrc/emcs-tfe/blob/ETFE-2533/app/uk/gov/hmrc/emcstfe/models/request/SoapEnvelope.scala#L64-L66 for completeness?

Comment (LewisMitchell) [reply app/uk/gov/hmrc/emcstfe/models/request/SubmitCreateMovementRequest.scala:89]: I could do that, would we set the message identifier for ChRIS only in the CAM flow? And the rest will be messageUUID?

Comment (grahampaulcook) [reply app/uk/gov/hmrc/emcstfe/models/request/SubmitCreateMovementRequest.scala:89]: Yes I believe that's the best approach and mirror's legacy.

Comment (LewisMitchell) [reply app/uk/gov/hmrc/emcstfe/models/request/SubmitCreateMovementRequest.scala:90]: Done in [6f2b939](https://github.com/hmrc/emcs-tfe/pull/96/commits/6f2b939ed4983fc8b3c058de67a887a5dcb12f16)

**Review by grahampaulcook — APPROVED** [review]

**Pull request merged**